seedfarmer/commands/_stack_commands.py

```diff
@@ -607,14 +624,25 @@ def deploy_seedkit(
         _logger.debug("SeedKit exists and not updating for Account/Region: %s/%s", account_id, region)
     else:
         _logger.debug("Initializing / Updating SeedKit for Account/Region: %s/%s", account_id, region)
-        commands.deploy_seedkit(
-            seedkit_name=config.PROJECT,
-            deploy_codeartifact=deploy_codeartifact,
-            session=session,
-            vpc_id=vpc_id,
-            subnet_ids=private_subnet_ids,
-            security_group_ids=security_group_ids,
-        )
+
+        seedkit_args = {
+            "seedkit_name": config.PROJECT,
+            "deploy_codeartifact": deploy_codeartifact,
+            "session": session,
+            "vpc_id": vpc_id,
+            "subnet_ids": private_subnet_ids,
+            "security_group_ids": security_group_ids,
+        }
+
+        if version.parse(cs.__version__) >= version.parse("1.3.0"):
```

Comment:
attention: since codeseeder release was cut, we can now remove this. I am not saying we should though, this does not hurt and will only prevent passing new args to older versions